### PR TITLE
Creates docs folder and first adr.

### PR DESCRIPTION
**Pull request by johanna-skylight**
## [No Ticket] Introduces decision records to project documentation 

Architectural decision records (adr) are a simple standard to document decisions and changes in the project that impact its architecture. The team agreed to start this record log for SimpleReport during the engineering sync of 04/20/2022. The Flexible DB migration is the first architectural change that we will record with this method.

## Changes Proposed

- Create docs folder at the root of the project and the first architectural decision record for SimpleReport.

## Additional Information
- A docs folder and a markdown file located in the frontend project was removed as it seemed that was created by create react app and never customized or used for the simple report project.

### CODE REVIEW

File: docs/architecture/decisions/flexible-db-migration.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+Movement to the Flexible Server SKU would provide a number of advantages, many of which are covered in [this documentation](https://docs.microsoft.com/en-us/azure/postgresql/flexible-server/overview). Specifically, the following provide the greatest impact to SimpleReport:
+- High availability of database instances, ensuring minimal downtime in the event of a datacenter or machine outage.
+- Automated patching with a managed maintenance window, ensuring that Azure does not attempt to perform maintenance of our DB during peak SR usage hours
+- Rapid scaling and performance management, enabling rapid response to customer demand
+- The integration of PgBouncer connection pooling, which increases resource efficiency and minimizes the need for manual connection management
+
+## Consequences
+
+The Flexible DB rollout has helped reduce response time according to the Azure metrics.
```
Comment (johanna-skylight) [inline docs/architecture/decisions/flexible-db-migration.md:28]: @emmastephenson I tried to capture the improvement in response times that you mentioned on the engineering channel as positive consequences. Let me know if it looks good or you have any suggestions on the wording or details.

**Review by nickclyde — APPROVED** [review]
Thanks so much for adding this Johanna!

**Review by zdeveloper — APPROVED** [review]
LGTM! Thank you!

File: docs/architecture/decisions/flexible-db-migration.md (hunk 1 above)
Comment (emmastephenson) [reply docs/architecture/decisions/flexible-db-migration.md:28]: This looks fantastic, thank you so much!

**Review by emmastephenson — APPROVED** [review]

**Pull request merged**